packages/design-tokens/src/tokens/functional/components/testimonial/base.json

```diff
@@ -0,0 +1,15 @@
+{
+    "brand": {
+      "Testimonial": {
+        "quoteSize": {
+          "default": {
+            "value": "var(--brand-text-size-300)"
+          },
+          "large": {
+            "value": "var(--brand-text-size-700)"
```

Comment:
👋 @mperrotti this is currently setting at 48px, but it should be the new 40px we've introduced in global typography refactoring recently. Could we update to use those styles? There are other associated styles we would also want to set here, like line-height. For reference 👇 

```
.Heading--4 {
  font-weight: var(--brand-heading-weight-600);
  font-size: var(--brand-text-size-600);
  line-height: var(--brand-text-lineHeight-600);
  font-family: var(--brand-heading-fontFamily);
  letter-spacing: var(--brand-heading-letterSpacing-600);
}
```